src/Microsoft.VisualStudio.Editors/Resources/Microsoft.VisualStudio.Editors.Designer.resx

```diff
@@ -1970,6 +1970,18 @@ Do you want to update the value in the .settings file?</value>
   <data name="PPG_AdvancedBuildSettings_DebugInfo_Portable" xml:space="preserve">
     <value>Portable</value>
   </data>
+  <data name="PPG_BuildSettings_Nullable_Disable" xml:space="preserve">
+    <value>Disable</value>
+  </data>
+  <data name="PPG_BuildSettings_Nullable_Enable" xml:space="preserve">
+    <value>Enable</value>
+  </data>
+  <data name="PPG_BuildSettings_Nullable_Warnings" xml:space="preserve">
+    <value>Warnings</value>
+  </data>
+  <data name="PPG_BuildSettings_Nullable_Annotations" xml:space="preserve">
+    <value>Annotations</value>
+  </data>
```

Comment:
By placing these values in .resx file we're choosing to localize them. Is that what you want, or would it make more sense to simply use the literal values understood by the compiler?

In the case of the latter you would hard-code them in the source and use the lower-case forms.

Reply:
The drop down should be localized, what we persist in the project file shouldn't be.

Reply:
I considered using the compiler's strings and ended up following the example set by the advanced build properties page (where I initially added this new UI before moving it to the regular build properties page).

![image](https://user-images.githubusercontent.com/350947/72856758-a0cfd400-3d0f-11ea-9392-65fa919936fb.png)

![image](https://user-images.githubusercontent.com/350947/72856742-98779900-3d0f-11ea-8e33-4c6aaa3b021c.png)

Reply:
Yes, and these were localized in response to a bug.